browser/BUILD.gn

```diff
@@ -304,6 +304,10 @@ source_set("browser_process") {
     sources += [
```

Comment:
we need to start cleaning this stuff up so can you follow the pattern in brave/browser/sources.gni and move this into browser/ipfs/sources.gni ?

Reply:
please be sure to include all necessary deps there as well

Reply:
adding files to browser_process is deprecated. cc @mariospr can you add a comment in one of your PRs to that effect? Likely no one will see it because that target is ridiculously large, but at least it will be there

Reply:
actually I should just get https://github.com/brave/brave-core/pull/8538 reviewed so you don't have to

Reply:
I'm about to get https://github.com/brave/brave-core/pull/8538 into land-able state, so maybe we could wait on that before landing this one if it makes sense? Or we could do it the other way around too, shouldn't be too hard either.